Kernel/Firmware/ACPI/Parser.cpp

```diff
@@ -205,6 +206,19 @@ UNMAP_AFTER_INIT void Parser::process_fadt_data()
     m_hardware_flags.wbinvd_flush = (sdt->flags & (u32)FADTFlags::FeatureFlags::WBINVD_FLUSH);
 }
 
+UNMAP_AFTER_INIT void Parser::process_dsdt()
+{
+    auto sdt = Memory::map_typed<Structures::FADT>(m_fadt).release_value_but_fixme_should_propagate_errors();
+
+    m_sdt_pointers.append(PhysicalAddress(sdt->dsdt_ptr));
+    auto dsdt_or_error = Memory::map_typed<Structures::DSDT>(PhysicalAddress(sdt->dsdt_ptr));
```

Comment:
What about this line?
https://github.com/SerenityOS/serenity/blob/a1943039489b1e9c4742498ee8424e10379de5a4/Kernel/Firmware/ACPI/Parser.cpp#L173

Reply:
> What about this line?
> 
> https://github.com/SerenityOS/serenity/blob/a1943039489b1e9c4742498ee8424e10379de5a4/Kernel/Firmware/ACPI/Parser.cpp#L173

Other ACPI tables print their physical address so I did the same

Reply:
What I mean is that it will print twice the same line for no good reason.

Reply:
I removed the printout in process_fadt_data and extended the message in process_dsdt